finout/README.md

```diff
@@ -0,0 +1,41 @@
+# Finout: The Modern cost Observability platform for FinOps
+
+## Overview
+
+
+[Finout](https://finout.io) combines Cloud Providers, Data Warehouses, and CDNs  
+into one mega bill, enabling an unparalleled business context view of your cloud spend with no heavy lifting in minutes.
+
+Finout’s cloud cost management tool uses your container metrics in order to break down Kubernetes costs without the need to install an agent. You can connect to any metric within Datadog and use it to create a unit cost, saving countless hours and improving your cost observability company-wide.
+
+Finout’s Datadog application allows you to add cost widgets into your Datadog dashboards, combining your AWS, Snowflake, Kubernetes, and Unit costs alongside your usage in one place.
+
+## Setup
+
+### Installation
+
+If you don't already have a Finout account, [sign up](https://finout.io/start-now) for your free trial.
+Once you get the invitation to your account, you are all set to start working with the Finout Application in Datadog within minutes, no code needed. 
+
+To get a Kubernetes-level cost breakdown, or a unit cost that is based on a Datadog metric, please reach out to Finout support directly in the Finout application or by emailing us at [support@finout.io](mailto:support@finout.io).
+
+### Configuration
+
+1. Install the app from this tile
+2. Add the Finout widget to your dashboard
+3. Authenticate with your Finout account
+4. Configure the widget parameters
+	1. Usage type (Cost, Total Coverage, RI, SP, Spot, or On-Demand)
+	2. Chart type (Bar or Line)
+	3. Context (Choose the Finout business context you want to visualize)
+	4. View (Which Finout saved view do you want to visualize)
+5. Add as many Finout widgets as you like!
+
+## Troubleshooting
+
+Please note that when looking at AWS cost you may need to expand the timeframe of your query to at least a week, as AWS cost and usage reporting is always two days behind.
```

Comment:
```suggestion
**Note:** When looking at AWS cost, you may need to expand the timeframe of your query to at least one week because AWS cost and usage reporting are always two days behind.
```